CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationPointsOfInterest.uc

```diff
@@ -0,0 +1,199 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    X2StrategyElement_InfiltrationPointsOfInterest.uc
+//  AUTHOR:  NotSoLoneWolf
+//  PURPOSE: Create new POI templates needed for the mod
+//---------------------------------------------------------------------------------------
+
+class X2StrategyElement_InfiltrationPointsOfInterest extends X2StrategyElement
+	dependson(X2PointOfInterestTemplate);
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> Templates;
+	
+	Templates.AddItem(CreatePOIPrototypeT2Template());
+	Templates.AddItem(CreatePOIPrototypeT3Template());
+	Templates.AddItem(CreatePOISidegradeT1Template());
+	Templates.AddItem(CreatePOISidegradeT2Template());
+	Templates.AddItem(CreatePOISidegradeT3Template());
+
+	//Templates.AddItem(CreatePOIReduceRelayCostTemplate());
+
+	return Templates;
+}
+
+//---------------------------------------------------------------------------------------
+
+static function X2DataTemplate CreatePOIPrototypeT2Template()
+{
+	local X2PointOfInterestTemplate Template;
+
+	`CREATE_X2POINTOFINTEREST_TEMPLATE(Template, 'POI_Prototype_T2');
+
+	Template.CanAppearFn = CurrentTierFully1;
+
+	return Template;
+}
+
+static function X2DataTemplate CreatePOIPrototypeT3Template()
+{
+	local X2PointOfInterestTemplate Template;
+
+	`CREATE_X2POINTOFINTEREST_TEMPLATE(Template, 'POI_Prototype_T3');
+
+	Template.CanAppearFn = CurrentTierFully2;
+
+	return Template;
+}
+
+static function X2DataTemplate CreatePOISidegradeT1Template()
+{
+	local X2PointOfInterestTemplate Template;
+
+	`CREATE_X2POINTOFINTEREST_TEMPLATE(Template, 'POI_Sidegrade_T1');
+
+	Template.CanAppearFn = CurrentTierFully1;
+
+	return Template;
+}
+
+static function X2DataTemplate CreatePOISidegradeT2Template()
+{
+	local X2PointOfInterestTemplate Template;
+
+	`CREATE_X2POINTOFINTEREST_TEMPLATE(Template, 'POI_Sidegrade_T2');
+
+	Template.CanAppearFn = CurrentTierPartially2;
+
+	return Template;
+}
+
+static function X2DataTemplate CreatePOISidegradeT3Template()
+{
+	local X2PointOfInterestTemplate Template;
+
+	`CREATE_X2POINTOFINTEREST_TEMPLATE(Template, 'POI_Sidegrade_T3');
+
+	Template.CanAppearFn = CurrentTierPartially3;
+
+	return Template;
+}
+/*
+static function X2DataTemplate CreatePOIReduceRelayCostTemplate()
+{
+	local X2PointOfInterestTemplate Template;
+
+	`CREATE_X2POINTOFINTEREST_TEMPLATE(Template, 'POI_ReduceRelayCost');
+
+	Template.CanAppearFn = CanBuildRadioRelay;
+
+	return Template;
+}
+*/
+//----------------------------------------------------------------------
+
+static function bool CurrentTierPartially1(XComGameState_PointOfInterest POIState)
+{
+	if (`ONLINEEVENTMGR.HasTLEEntitlement())
```

Comment:
I don't think this check(s) is needed as the mod now fully depends on TLE anyway. https://github.com/WOTCStrategyOverhaul/CovertInfiltration/issues/336

Reply:
Oh, I thought what that issue meant was "make sure stuff is compatible with non-TLE users". Guess I didn't read the title

Reply:
Yeah, I don't think it's worth the effort to build the checks into the main mod, especially since many parts come from config